### ISSUE TITLE
Compile fails on Windows (CMake Files target??)

### ISSUE THREAD

**Issue by msarahan** [Blosc/bcolz#268]
I'm trying to compile with VS 2015, setuptools 18.4.  I have compiled c-Blosc externally, and I'd like to just use it.  I'm missing part of the build process:

```
C:\Users\builder\MC_Main\conda-bld\work>python setup.py build_ext --inplace --blosc=C:\Users\builder\MC_Main\envs\_build\Library install
running build_ext
LINK : fatal error LNK1181: cannot open input file 'Files.obj'
error: Command "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64\link.exe /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:C:\Users\builder\MC_Main\envs\_build\Library\lib /LIBPATH:C:\Users\builder\MC_Main\envs\_build\libs /LIBPATH:C:\Users\builder\MC_Main\envs\_build\PCbuild\amd64 /LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\LIB\amd64 /LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\LIB\amd64 /LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.10240.0\ucrt\x64 /LIBPATH:C:\Program Files (x86)\Windows Kits\NETFXSDK\4.6\lib\um\x64 /LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.10240.0\um\x64 /LIBPATH:C:\Users\builder\MC_Main\envs\_build\Library\lib /LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\LIB\amd64 /LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\LIB\amd64 /LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.10240.0\ucrt\x64 /LIBPATH:C:\Program Files (x86)\Windows Kits\NETFXSDK\4.6\lib\um\x64 /LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.10240.0\um\x64 blosc.lib /EXPORT:PyInit_carray_ext build\temp.win-amd64-3.5\Release\bcolz\carray_ext.obj /OUT:build\lib.win-amd64-3.5\bcolz\carray_ext.cp35-win_amd64.pyd /IMPLIB:build\temp.win-amd64-3.5\Release\bcolz\carray_ext.cp35-win_amd64.lib" failed with exit status 1181
```

I have narrowed this down to (possibly) a build target in the internal c-blosc CMakeLists, but don't understand why this is showing up in building bcolz with the external c-blosc.  This error is actually why I tried building Blosc externally in the first place.

I have thought that maybe setuptools find_packages was responsible, but I tried excluding the c-blosc from find_packages, and nothing changed.

I can't find any explicit mention of the Files target in the carray_ext.pyx file.  Is there something I'm missing here?

**Comment by cgohlke** [comment]
The LIBPATHs are not quoted. The "Files" target comes from unquoted "Program Files". See also http://bugs.python.org/issue13765 and https://github.com/numpy/numpy/issues/2611.

To work around, remove the `BuildExtNumpyInc` build_ext cmdclass in setup.py and use `inc_dirs = ['bcolz', numpy.get_include()]`.

**Comment by msarahan** [comment]
Thanks, Christoph.  I dont' think I would have figured this one out on my own.  Kudos to Almar for originally figuring this out.

**Comment by FrancescAlted** [comment]
Addressed in commit 0467782.  Thanks for the report and the fix!

**Comment by mindw** [comment]
This should be resolved in numpy/numpy#5765
